przezdziecki/CenteringThings

mat-px opened the pull request:
-Added a parameter to center a lot of things into a couple css files
-Turned officer page into columns
-Fixed 404page css loading issue

mat-px commented:
Although the officers are centered, it still looks off because of the names.
-Need to either change to column style or shift the boxes right

joe-studer-18 commented on active/views/abouttabs/Officers.html:
```diff
@@ -127,15 +127,14 @@ <h4>BA of KA Officers</h4>
       <section id="officers">
         <div class="about-container">
-          <div class="about-section-heading">
+          <div class="about-section-heading" style="text-align: center;">
```
Can you add the style here into the about-section-heading class instead of setting it inline?

mat-px replied:
I can make a different class if you want, but all of the officer names use the same about-section-heading class, which don't want to be centered, unlike the title.

joe-studer-18 commented on active/views/abouttabs/Officers.html:
```diff
@@ -337,8 +310,7 @@ <h3> Sam George </h3>
 
-        <div class="about-container">
-
+        <div class="about-container-officers" style="justify-content: left;padding-left: 50px;">
```
Same as above, can you add the inline style to the existing class, or create a new one instead?

joe-studer-18 commented on active/views/valuetab/goKA.html:
```diff
@@ -125,7 +125,7 @@ <h4>Top Five Reasons to Go KA</h4>
     <section id="go-KA">
         <div class="values-container">
-            <div class="values-listing">
+            <div class="values-listing" style="justify-content: left;">
```
Same deal w/ the inline declarations.

joe-studer-18 replied:
Looks like you missed a few inline css :)

joe-studer-18 reviewed (commented):
Also, not sure if it is already, but can you push this to staging?

joe-studer-18 commented:
I'd probably recommend trying a [grid](https://css-tricks.com/snippets/css/complete-guide-grid/) or using [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). Grid will be really nice for desktop, but could become a pain with swapping to mobile since it's a bit more static in its design. Flex will take a bit more initial set up, but should be able to be a bit more flexible in spacing.

joe-studer-18 commented on active/404page.html:
```diff
@@ -1,107 +1,109 @@
 <!DOCTYPE html>
```
Not sure how to do file comments in git, but can you move this to a separate PR? It's best to keep these kind of things in separate merges in case things break and we need to roll back.

mat-px commented:
-I reverted the 404page and made another PR
-I changed the style of the officer page to columns so it would be easier for mobile, let me know if you liked it better as rows.
-Put everything back into css, I think

joe-studer-18 commented on active/style/about.css:
```diff
@@ -82,30 +93,33 @@ body #greek-life, #house-history, #letter-ka-parent, #notable-alumni, #officers,
+  margin-bottom: 50px;
+}
+@media screen and (max-width: 500px) {
+  img {
+         width: 70%;
+    }
```
nit: can you fix the formatting of this block

joe-studer-18 commented on active/views/abouttabs/Officers.html:
```diff
@@ -134,238 +134,235 @@ <h2> BA of KA Officers </h2>
+                President
               <br> <a href="mailto:tahg47@umsystem.edu">twbh3@umsystem.edu</a>
               <br> Year Joined KA: 2018
+            </div>
```
nit: Can you fix the formatting of these lines? That </p> below should be in line with the one on 145, and everything w/in should be indented as normal.

joe-studer-18 replied:
*And for all the ones below

joe-studer-18 reviewed (commented):
Did you push to devel or staging? I'm not seeing anything different when I check out the officer page on those.

joe-studer-18 reviewed (approved):
This looks great, thanks for doing this!
